### PR TITLE
Features: swaps by favorite tokens, swaps portfolio 

### PR DESCRIPTION
**Pull request by OgnjenCBS**

### CODE REVIEW

File: src/modules/swaps/swaps.repository.ts [hunk 1]
```diff
@@ -37,4 +37,48 @@ export class SwapRepository {
...
+      where: [{ inputAssetId: In(assetIds) }, { outputAssetId: In(assetIds) }],
+    });
+
+    let swaps: SwapDto[] = [];
+
+    const meta = new PageMetaDto(pageOptions.page, pageOptions.size, count);
+
+    data.forEach((swap) => {
+      swaps.push(this.swapMapper.toDto(swap));
+    });
```
Comment (Str4LeGit) [inline src/modules/swaps/swaps.repository.ts:58]: There is not need to loop, there is `.toDtos` function in BaseMapper that handles arrays.

File: src/modules/swaps/swaps.repository.ts [hunk 2]
```diff
@@ -37,4 +37,48 @@ export class SwapRepository {
...
+      where: { accountId: accountId },
+    });
+
+    let swaps: SwapDto[] = [];
+
+    const meta = new PageMetaDto(pageOptions.page, pageOptions.size, count);
+
+    data.forEach((swap) => {
+      swaps.push(this.swapMapper.toDto(swap));
+    });
```
Comment (Str4LeGit) [inline src/modules/swaps/swaps.repository.ts:80]: same as above

File: src/modules/swaps/swaps.repository.ts [hunk 3]
```diff
@@ -37,4 +37,48 @@ export class SwapRepository {
 
     return new PageDto(swaps, meta);
   }
+
+  async findSwapsByAssetIds(
```
Comment (Str4LeGit) [inline src/modules/swaps/swaps.repository.ts:41]: I think that with this function, the one above (findSwapsByAssetId) can be removed. Instead you can invoke this one with a single element in array.

Comment (OgnjenCBS) [reply src/modules/swaps/swaps.repository.ts:41]: Which means that function findSwapsByAssetId in repository could also be deleted?

Comment (DanijelCBS) [reply src/modules/swaps/swaps.repository.ts:19]: The old endpoint should stay until frontend and apps are updated with new one.

**Pull request merged**